File: sample-app/src/main/java/io/github/lazymockbean/sampleapp/service/SampleServiceB.java
package io.github.lazymockbean.sampleapp.service;

import org.springframework.stereotype.Service;

@Service
public class SampleServiceB {
    public String getSample1() {
        return "sample-B-1";
    }

    public String getSample2() {
        return "sample-B-2";
    }
}

File: sample-app/src/main/kotlin/io/github/taesk/lazymockbean/sampleapp/service/SampleServiceB.kt
package io.github.taesk.lazymockbean.sampleapp.service

import org.springframework.stereotype.Service

@Service
class SampleServiceB {
    fun getSample1(): String {
        return "sample-B-1"
    }

    fun getSample2(): String {
        return "sample-B-2"
    }
}
